orga/tests/acceptance/campaign-activity_test.js

```diff
@@ -76,13 +76,14 @@ module('Acceptance | Campaign Activity', function (hooks) {
   module('when prescriber reset filters', () => {
     test('should reset status filter', async function (assert) {
       // when
-      await visit('/campagnes/1');
+      const screen = await visit('/campagnes/1');
 
       await fillByLabel('Statut', 'STARTED');
       await clickByName('Effacer les filtres');
 
       // then
-      assert.equal(find('[aria-label="Statut"]').selectedOptions[0].value, '');
+      const element = await screen.findByLabelText('Statut');
+      assert.equal(element.value, '');
```

Comment:
**Question**
Est-ce qu'il y a besoin d'`await` le findByLabelText ? 
Dans certains fichier de test j'ai déjà vu : `assert.dom(screen.getByLabelText('Pix+Droit')).isChecked();` par ex.
Je me demandais si ici on ne pouvait pas aussi écrire quelque chose du genre : 
```
assert.equal(screen.findByLabelText('Statut').value, '');
```

Reply:
Oui parce que les fonctions findBy retourne une promesse, dans la doc de testing library
"The findBy and findAllBy queries are async and retry until the query returns successfully, or when the query times out; they wrap waitForElement"
Mais en effet on pourrait utiliser un get dans ce cas puisqu'on sait que l'element existe

Reply:
J'ai pu le modifier au getByLabelText 😄